src/development/packages-and-plugins/developing-packages.md

```diff
@@ -614,6 +614,15 @@ a section in [Supporting the new Android plugins APIs][].
 If you want to develop a package that calls into native APIs using
 Dart's FFI, you need to develop an FFI plugin package.
 
+Both FFI plugin packages and (non-FFI) plugin packages support
+bundling native code, but FFI plugin packages do not support
+method channels and do include method channel registration code.
+If you want to implement a plugin that uses both method channels
+and FFI, use a (non-FFI) plugin. You can chose this per platform.
```

Comment:
The last sentence probably needs a bit of rework to make it accessible to all our developers. /cc @sfshaza2